ONSdigital/eq-author-app
# Added check that validation was missing before recreating it

> Describe what you have changed as part of this task and why; include a link to the Trello card/GitHub issue, related pull requests, etc as required.
>
> Added check that Validation was missing before recreating it thereby wiping out existing validation value

### How to review 

* create a survey
* add three answers of same type such as number or percentage
* add a validation total
* remove any one of the answers and the total should remain
* add an answer of a different type and the total will become unavailable
* remove the new answer leaving all remaining with same answer type
* total should be re-enabled but cleared of previous value

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Trello card for this task into the next stage of the process
